### PR TITLE
test: fix failing DAPI Platform tests

### PR DESCRIPTION
**Pull request by markin-io**
## Issue being fixed or feature implemented

Fixed failing Platform tests 

## What was done?

- Updated DAPI Client and Wallet lib to latest version
- Increased timeout for sendTransaction precondition in quorum test

## How Has This Been Tested?

Locally on testnet `./bin/test testnet`

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: test/smoke/quorum.js [hunk 1]
```diff
@@ -216,8 +216,9 @@ describe('Quorums', () => {
       return Promise.all(promises);
     });
 
-    before('Send a transaction', async () => {
-      const timeout = 25000; // set individual rpc client timeout
+    before('Send a transaction', async function before() {
+      const timeout = 60000; // set individual rpc client timeout
```
Comment (markin-io) [inline test/smoke/quorum.js:220]: @ktechmidas please advise on value. With 25s it times out. Tried 40s, but it's flaky

Comment (shumkov) [reply test/smoke/quorum.js:220]: Which network?

Comment (ktechmidas) [reply test/smoke/quorum.js:220]: Yeah this is fine, 15 was always too low

Comment (ktechmidas) [reply test/smoke/quorum.js:220]: testnet @shumkov

**Review by ktechmidas — APPROVED** [review]
LGTM

**Pull request merged**